libraries/botbuilder-dialogs/botbuilder/dialogs/dialog_extensions.py

```diff
@@ -75,3 +81,9 @@ class DialogExtensions:
             results = await dialog_context.continue_dialog()
             if results.status == DialogTurnStatus.Empty:
                 await dialog_context.begin_dialog(dialog.id)
+
+    @staticmethod
+    def __is_eoc_coming_from_parent(turn_context: TurnContext) -> bool:
+        # To determine the direction we check callerId property which is set to the parent bot
+        # by the BotFrameworkHttpClient on outgoing requests.
+        return not turn_context.activity.caller_id
```

Comment:
I believe the return value should be:
`return bool(turn_context.activity.caller_id)`